lib/store/settings_store.dart

```diff
@@ -587,6 +597,9 @@ abstract class SettingsStoreBase with Store {
   @observable
   SeedType moneroSeedType;
 
+  @observable
+  DerivationTypeSetting bitcoinDerivationType;
```

Comment:
**Minor:**
would be easier to rename it to bitcoinSeedType (everywhere)

Reply:
But it it does not change the seed type, but only the key derivation